### PR TITLE
Make WASM path configurable + Update IFC library

### PR DESCRIPTION
**Pull request by agviegas**
Related issue: #21673

**Description**

This solves the route to the WASM file as a setter of IfcLoader, so the end user can put it wherever they prefer. Also, I have updated the web-ifc library to the latest version.

### CODE REVIEW

**Comment by agviegas** [comment]
You can find a working example [here](https://github.com/agviegas/ifcjs-hello-world) (see [app.js](https://github.com/agviegas/ifcjs-hello-world/blob/main/app.js)).

**Comment by Mugen87** [comment]
It seems the example is not working anymore:

https://raw.githack.com/agviegas/three.js/IfcLoader/examples/webgl_loader_ifc.html

> Uncaught SyntaxError: Unexpected token ')'

File: examples/jsm/loaders/IFCLoader.js [hunk 1]
```diff
@@ -171,6 +171,9 @@ class IFCLoader extends Loader {
 
 	}
 
-}
+	setWasmPath(path){
+		ifcAPI.SetWasmPath(path);
+	}
+} );
```
Comment (Mugen87) [inline examples/jsm/loaders/IFCLoader.js:177]: I guess this line should be just:
```js
}
```

**Comment by agviegas** [comment]
It is working on my end now. Let me know if there is anything else to correct!

**Comment by mrdoob** [comment]
Thanks!

**Comment by mrdoob** [comment]
The tree is no longer red (and e2e didn't caught this...) 🤔

Screenshot 2021-04-20 at 08 11 39

**Comment by agviegas** [comment]
Hum, probably the trees are not red anymore because of the last advances regarding IFC material reading (I think that red was a default case). If it's not urgent, I can update the shot of the example in the next PR (which will be soon).

**Comment by mrdoob** [comment]
No worries! I'll update the screenshot now.

**Comment by mrdoob** [comment]
https://github.com/mrdoob/three.js/commit/be73dd582a7daa912e02f14e6968d6a3f87a6aba

**Pull request merged**